app/src/test/java/com/kks/nimblesurveyjetpackcompose/viewmodel/survey/SurveyDetailViewModelTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.kks.nimblesurveyjetpackcompose.viewmodel.survey
+
+import com.kks.nimblesurveyjetpackcompose.base.BaseViewModelTest
+import com.kks.nimblesurveyjetpackcompose.model.ResourceState
+import com.kks.nimblesurveyjetpackcompose.model.SurveyQuestion
+import com.kks.nimblesurveyjetpackcompose.repo.survey.SurveyRepo
+import com.kks.nimblesurveyjetpackcompose.surveyQuestions
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.advanceUntilIdle
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.*
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class SurveyDetailViewModelTest : BaseViewModelTest() {
+    private lateinit var viewModel: SurveyDetailViewModel
+    private val surveyRepo: SurveyRepo = mockk()
+
+    override fun setup() {
+        super.setup()
+        viewModel = SurveyDetailViewModel(surveyRepo = surveyRepo, ioDispatcher = testDispatcher)
+    }
+
+    @Test
+    fun `When get survey questions and return error, show error`()= runTest {
```

Comment:
We should reformat the whole file. 😄 
```suggestion
    fun `When get survey questions and return error, show error`() = runTest {
```